coralproject/talk
# It needed to have the Moderation Modal inside a tab container too.

## What does this PR do?

It resolves bug Issue #186
Adds a tab for the Shortcuts Modal. 

## How do I test this PR?

- go to admin
- click on shortcuts
- look at the dev console. There should be no error.
